failgood/jvm/src/failgood/ContextDSL.kt

```diff
@@ -110,4 +112,6 @@ interface ContextDSL<GivenType> : ResourcesDSL {
      */
     suspend fun test(name: String, tags: Set<String> = setOf(), function: TestLambda<GivenType>) =
         it(name, tags, function)
+
+    suspend fun describeSteps(name: String, function: suspend It<Unit>.() -> Unit) {}
```

Comment:
Accidental empty default implementation? 🤔

Reply:
just to flesh out the api